### DISCUSSION TITLE
I only saw this on Twitter and wanted to share it with you.

### DISCUSSION THREAD

**Discussion by voku** [mnapoli/simple-s3#4] [General]
https://gist.github.com/marcoarment/344d71e91d6cd3df5fe6db2ac08ff99f

**Comment by mnapoli** [comment]
Hey @voku, thanks!

This is actually exactly why I started this repo. The author said on Twitter they are not interested in adding it to Packagist, and it could be useful for Bref.

But I spent some time trying to actually use (and cleanup) this class, it just doesn't work. Like, at all. I'm not sure what happened here, there are some very obvious things that just don't run at all, missing code, etc.

So instead of pushing everything yesterday evening, I ended up the rabbithole of writing it from scratch, and it's taking much longer than I wanted 😅  Hopefully I can push that in the next few days.

**Comment by voku** [comment]
:smile: understand, thanks for your effort.

**Comment by mnapoli** [comment]
Done! You can see my iterations in the history ^^ https://github.com/mnapoli/simple-s3/commits/main

I need to clean it up a bit more but it's working for me right now.
